src/test/java/uk/gov/pay/products/validations/GatewayAccountRequestValidatorTest.java

```diff
@@ -75,4 +75,20 @@ public void shouldError_whenPathFieldIsMissing() {
         assertThat(errors.isPresent(), is(true));
         assertThat(errors.get().getErrors().toString(), is("[Field [path] is required]"));
     }
+
+    @Test
+    public void shouldError_whenEmptyFields() {
```

Comment:
I Suggest you add more unit tests here for all possible error messages. and Just have one Int test for 400